internal/query/projection/debug_notification_provider_test.go

```diff
@@ -37,7 +37,6 @@ func TestDebugNotificationProviderProjection_reduces(t *testing.T) {
 				aggregateType:    eventstore.AggregateType("instance"),
 				sequence:         15,
 				previousSequence: 10,
-				projection:       DebugNotificationProviderTable,
```

Comment:
the projections are removed because the want-statement already contains the projection, and used like this, the tests make no sense
instead, you want to check if the computed statement on the provided projection equals the want-statement(which already contains the want-projection)